ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml

```diff
@@ -80,6 +80,9 @@ tests:
     - as: test
       cli: latest
       commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      env:
+      - default: m5.4xlarge
+        name: CLUSTER_NODE_TYPE
```

Comment:
this PR merged before many of the rehearsals had completed, but I'd expect you'd need to also set some other environment variables to avoid clearing them.  See #14184 for similar recovery.  I have an internal ticket open with DPTP about rendering convenient docs for available environment variables [here][1] and similar.  Anyhow, if your CI is doing what you want, you can ignore this.  But if the jobs start behaving strangely, this may be why (and next time you may want to wait out the rehearsals and review their success before merging the release PR ;).

[1]: https://steps.ci.openshift.org/workflow/openshift-e2e-aws-hosted-loki

Reply:
> this PR merged before many of the rehearsals had completed, but I'd expect you'd need to also set some other environment variables to avoid clearing them. 

Do we have any idea what might need to be set or is this simply trial and error?